src/test/java/de/unistuttgart/iste/gits/reward/service/calculation/HealthScoreCalculatorTest.java

```diff
@@ -240,5 +240,20 @@ private AllRewardScoresEntity createAllRewardScoresEntityWithHealthOf(int health
                 .power(RewardScoreEntity.builder().value(0).build())
                 .build();
     }
+    @Test
+    void testCalculateInitialHealthValueForNewEntity() {
+        // Create a list of contents with different overdue days
+        List<Content> contents = new ArrayList<>();
+        contents.add(createContentWithUserData(UUID.randomUUID(), UserProgressData.builder().build(), 1));
+        contents.add(createContentWithUserData(UUID.randomUUID(), UserProgressData.builder().build(), 5));
+        contents.add(createContentWithUserData(UUID.randomUUID(), UserProgressData.builder().build(), 10));
+
+        // Calculate the initial health value using the method
+        int initialHealthValue = healthScoreCalculator.calculateInitialHealthValueForNewEntity(contents);
 
+        // Calculate expected initial health value based on your logic
+        int expectedInitialHealthValue = 100 - healthScoreCalculator.calculateHealthDecrease(contents, OffsetDateTime.now());
```

Comment:
please put in the actual value here that you expect.

Right now this is testing nothing, because you use the same function in the actual code.

So if the function "calculate Healthdecrease" has an error in it, this test cannot find it